starter-code/lab-mongoose-movies/routes/index.js

```diff
@@ -0,0 +1,93 @@
+const express = require('express');
+const router  = express.Router();
+const Celebrity = require ("../models/celebrity-model.js");
+
+/* GET home page */
+router.get('/', (req, res, next) => {
+  res.render('index');
+});
+
+router.get("/celebrities", (req, res, next) => {
+  Celebrity.find()
+  .then((celebritiesFromDb) => {
+    console.log(celebritiesFromDb);
+    res.locals.celebrityList =  celebritiesFromDb;
+    res.render("celebrities-page");
+  })
+  .catch((err) => {
+     next(err);
+  });
+  
+});
+
+router.get("/celebrities/new", (req, res, next) => {
+    res.render("new")
+})
+
+router.post("/process-celebrity", (req, res, next) => {
+  const {name, occupation, catchPhrase} = req.body;
+  Celebrity.create({name, occupation, catchPhrase})
+    .then (() => {
+      //redirect ONLY to URLs
+      // (redirect so we dont get duplicate data on refresh)
+       res.redirect("/celebrities");
+    })
+    .catch ((err) => {
+       next(err);
+    });
+});
+
+ router.get("celebrities/:celebrityId/edit", (req, res, next) => {
```

Comment:
It's really annoying, but you are simply missing a `/`. Should be 
``router.get('/celebrities/:celebrityId/edit', ... ``